kube-derive/tests/crd_schema_test.rs

```diff
@@ -132,7 +133,7 @@ fn test_serialized_matches_expected() {
             nullable: None,
             nullable_skipped_with_default: None,
             nullable_with_default: None,
-            timestamp: DateTime::from_utc(NaiveDateTime::from_timestamp_opt(0, 0).unwrap(), Utc),
+            timestamp: TimeZone::from_utc_datetime(&Utc, &NaiveDateTime::from_timestamp_opt(0, 0).unwrap()),
```

Comment:
Out of curiosity, was this a deprecation warning or something?

Reply:
Yeah, my `cargo test` command warned me about this, so I fixed it in a separate commit ;). It's [deprecated](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.from_utc) since `0.4.27`